metrics-operator/main.go

```diff
@@ -167,7 +168,7 @@ func main() {
 	}
 
 	var webhookBuilder certwebhook.Builder
-	if !disableWebhook {
+	if !disableWebhook && env.CertManagerEnabled {
```

Comment:
I think setting `env.CertManagerEnabled` to false will completely deactivate the webhook builder, which will prevent the operator from working properly.
Can you double check this by executing the integration tests with the corresponding environment variable set to `false`?
In any case, the aim of the issue is to only deactivate the check for the secret containing the certificate to be available, but not to deactivate the webhooks altogether.
This is done by setting the certificate watcher here via the `SetCertificateWatcher()` function: https://github.dev/geoffrey1330/lifecycle-toolkit/blob/2330e853adb167a77ffdbff3d877e5b182afe899/metrics-operator/main.go#L233-L234

So I guess the best approach is to, if `env.CertManagerEnabled` is false, pass a no-op implementation of the `ICertificateWatcher` interface (https://github.com/keptn/lifecycle-toolkit/blob/6e0f08bdd890880fc1f648985f61da84b005143c/keptn-cert-manager/pkg/certificates/watcher.go#L27).

This applies for both the lifecycle and metrics-operator

Reply:
Alright I will look into this.

Reply:
I've made the fix. could you please review